2023-02-22 10:42:22,288 [MainThread  ] [INFO ] [root]  STARTING BROWSER: /home/jake/research/chromium/src/out/direct-handles-no-gc/chrome
2023-02-22 10:42:22,288 [MainThread  ] [INFO ] [root]  STARTING BROWSER: driver: /home/jake/research/chromium/src/out/direct-handles-no-gc/chromedriver
2023-02-22 10:42:22,288 [MainThread  ] [INFO ] [root]  STARTING BROWSER: args: --no-default-browser-check --disable-component-update --disable-sync --no-experiments --enable-benchmarking --disable-extensions --no-first-run --disable-background-timer-throttling --disable-renderer-backgrounding --window-size=1500,1000 --enable-logging --log-file=/home/jake/research/v8_handles_experiments/crossbench_no_gc/chrome_v112_chrome-direct-no-gc_default/speedometer_2.1/5/browser.chrome.log
2023-02-22 10:42:25,524 [MainThread  ] [INFO ] [root]  RUNNING STORY
2023-02-22 10:44:14,860 [MainThread  ] [WARNI] [urllib3.connectionpool]  Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f93eea19eb0>: Failed to establish a new connection: [Errno 111] Connection refused')': /session/d3dfdedd92be31fb35e84287537e51bf
2023-02-22 10:44:14,860 [MainThread  ] [WARNI] [urllib3.connectionpool]  Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f93eea19700>: Failed to establish a new connection: [Errno 111] Connection refused')': /session/d3dfdedd92be31fb35e84287537e51bf
2023-02-22 10:44:14,861 [MainThread  ] [WARNI] [urllib3.connectionpool]  Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f93eea194c0>: Failed to establish a new connection: [Errno 111] Connection refused')': /session/d3dfdedd92be31fb35e84287537e51bf
